integration/simple-deprecated-fields/simple.ts

```diff
@@ -32,7 +32,9 @@ export interface Child {
   name: string;
 }
 
-const baseSimple: object = { name: '', age: 0, testField: '', testNotDeprecated: '' };
+function createBaseSimple(): Simple {
+  return { name: '', age: 0, child: undefined, testField: '', testNotDeprecated: '' };
```

Comment:
Nice to see this creating the correct type with all fields and avoid the invalid cast that we had before!